.github/workflows/test.yaml

```diff
@@ -0,0 +1,55 @@
+name: test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - devel
+      - stable
+  workflow_dispatch:
+    inputs:
+      pytest_addopts:
+        description:
+          Extra options for pytest; use -vv for full details; see
+          https://docs.pytest.org/en/latest/example/simple.html#how-to-change-command-line-options-defaults
+        required: false
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You don't seem to be using this at all, so let's better not expose secrets just because.

```suggestion
```